### DISCUSSION TITLE
커밋 후 push 과정에서의 질문사항

### DISCUSSION THREAD

**Discussion by gyulhongsy** [HeoJiye/Github-Study#17] [질문 게시판] [answered]
<img width="441" alt="화면 캡처 2023-01-26 163250" src="https://user-images.githubusercontent.com/80159575/214780622-835917c4-2da0-456c-8a7d-e318a4fc7dec.png">

readme파일은 폴더에서 직접 메모장으로 열어 수정했습니다.
이후에 push까지 실행했는데요, 제 깃에서는 pull request 창이 새로 뜨지도 않고 변경 사항도 없어보여서 질문 드립니다..
원래 변경 사항이 보이지 않는건가요?
아니면 폴더에서 파일을 직접 찾아서 수정하는것이 아니고 깃에서 수정한 다음 push과정을 했어야 하나요??

**Comment by HeoJiye** [comment, marked as answer]
맞게 진행하셨습니다!

`gyulhongsy` 브랜치를 새로 파서 푸시했기 때문에, 깃허브 레포지터리에서 `gyulhongsy` 브랜치를 확인해보세요.
기본으로는 `main` 브랜치가 보였을 거에요.

![image](https://user-images.githubusercontent.com/78946499/214794449-db7f4443-1289-42b6-81c5-74c3020c38b5.png)

![image](https://user-images.githubusercontent.com/78946499/214794065-71a07d65-79de-4b96-adf2-cd858e83bacf.png)

**Reply by HeoJiye** [reply]
그런데 제출은 하셨네요? 어떻게 하셨나요?

**Reply by gyulhongsy** [reply]
제가 만든 npc-pair-gyulhongsy 레포지토리로 들어가면 수정된 사항이 안보여요.. 새로 만든 레포지토리에서 수정사항이 보여야하는거 아닌가요..??
<img width="434" alt="dfdf" src="https://user-images.githubusercontent.com/80159575/214807946-d1afd259-3a8d-47fd-b517-f45dcf6c238f.png">

**Reply by gyulhongsy** [reply]
제출은 하려고했다기보다는 지난주 커리 다시 보면서 push했더니 저렇게 변경된 것 같아요

**Reply by gyulhongsy** [reply]
제가 이후에 이것저것 해보니 원하는 결과를 얻은 것 같아요!! 답변감사드립니다 :)
